doc/gettingStarted/gettingStarted-1.0.md

```diff
@@ -22,7 +22,7 @@ InfiniTime doesn't handle daylight savings automatically, so make sure to set th
 
 ### Digital watch face
 
-![Digital watch face](ui/watchface.jpg)
+![Digital watch face](ui/watch face.jpg)
```

Comment:
I think the jpg filename wasn't renamed (and shouldn't be ;)

```suggestion
![Digital watch face](ui/watchface.jpg)
```

Reply:
Absolutely! One of the caveats of regex..

Reply:
maybe you want to revert this line as well?